docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+
+   ```bash
+   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+   ```
+
+4. Set up the `external-dns-management` component.
+
+- Create a DNSProvider CR. See the following example and modify values for the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters. As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     type: your-provider-dns-type # Edit this value
+     secretRef:
+       name: secret-name # Edit this value
+     domains:
+       include:
+         - mydomain.com # The main domain. Edit this value
+   EOF  
+   ```
+
+- Create a DNSEntry CR. See the following example and modify values for the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     dnsName: "my.sub1.mydomain.com" # Your subdomain. Edit this value
+     ttl: 600
+     targets:
+       - 1.2.3.4 # IP address. Edit this value
+   EOF
+   ```
+
+    >**NOTE:** To check the Ingress Gateway IP, run:
+    >```
+    >kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    >```
+
+   >**NOTE:** You can create many DNSEntry CRs for one DNSProvider, depending on the number of subdomains you want to use.
+
+5. Create an Issuer CR. See the following example and modify values of the **metadata.spec.acme.email**, **metadata.spec.domains.include**, and **metadata.spec.domains.exclude** parameters. As the values for the **metadata.spec.domains.include** parameter, use the main domain, the subdomain from the **spec.dnsName** parameter of the DNSEntry CR, and a wildcard DNS record. Run:
+
+   ```bash
```

Comment:
General note: It's very unconvenient for users to have a "looks-like-copy-paste" command that still requires manual edits. Users have to copy it, paste into some temporary notepad, edit, then copy and paste into terminal. IMO we should chose between:
- ask user to define some environment variables and use these in the command - then users have real copy-paste experience.
- provide an object template in YAML format (In this case this would be `Issuer`). User will save the template to the file, edit missing fields, and then manually apply the file. This requires a bit more work, but is more effective if you want to play with the tutorial, do it several times etc.